components/bio-formats-plugins/src/loci/plugins/out/Exporter.java

```diff
@@ -299,14 +300,22 @@ public void run() {
             catch (DependencyException de) { }
             catch (ServiceException se) { }
 
+            OMEXMLMetadataRoot root = (OMEXMLMetadataRoot) store.getRoot();
```

Comment:
This line needs to be moved after line 305, otherwise there is a risk of a NullPointerException.